file-service-persistence/pom.xml

```diff
@@ -15,14 +15,24 @@
         <dependency>
             <groupId>uk.gov.justice.services</groupId>
             <artifactId>persistence-jdbc</artifactId>
+            <version>${cpp.framework.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>uk.gov.justice.services</groupId>
+            <artifactId>file-service-api</artifactId>
             <version>${project.version}</version>
         </dependency>
         <dependency>
             <groupId>uk.gov.justice.services</groupId>
-            <artifactId>file-api</artifactId>
+            <artifactId>file-service-api</artifactId>
             <version>${project.version}</version>
         </dependency>
         <dependency>
+            <groupId>javax</groupId>
+            <artifactId>javaee-api</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
             <groupId>org.postgresql</groupId>
             <artifactId>postgresql</artifactId>
             <version>9.4.1208</version>
```

Comment:
It isn't in any of the parent poms - looking through the other projects, they all have the postegres version as a property. I'm guessing that's because the production code doesn't know which database it's talking to - the datasource is given by the container - so other projects just use postgres classes for test. 
However, I need to include postges as I'm using one of it's classes for inserting JsonB. 
Should we move the version to one of the parent poms?